engine/dlib/src/dlib/message.cpp

```diff
@@ -421,6 +422,13 @@ namespace dmMessage
         return RESULT_OK;
     }
 
+    Result Post(const URL* sender, const URL* receiver, dmhash_t message_id, uintptr_t user_data1, uintptr_t descriptor,
+                    const void* message_data, uint32_t message_data_size, MessageDestroyCallback destroy_callback)
+    {
+        return Post(sender, receiver, message_id, user_data1, 0, descriptor, message_data, message_data_size, destroy_callback);
+    }
```

Comment:
Keeping a legacy version, with only one userdata.
That way I don't have to change _all_ the places currently.